forms-web-app/src/lib/documents-api-wrapper.js

```diff
@@ -44,5 +44,11 @@ exports.createDocument = async (appeal, formData) => {
     throw new Error(apiResponse.statusText);
   }
 
-  return apiResponse.json();
+  const response = await apiResponse.json();
+
+  if (!response.id) {
```

Comment:
Is it worth adding some logging? I believe the `apiResponse` will log it anyway, but might make debugging easier

Something like `req.log.warn({ response }, 'Document had no ID')`

Reply:
sure 👍